CRD-DTR/DrugHasREMS/R4/resources/Questionnaire-R4-Prescriber-Enrollment.json

```diff
@@ -105,66 +105,54 @@
                 },
                 {
                     "linkId": "1.7",
-                    "text": "State License #",
-                    "type": "string",
-                    "required": true
-                },
```

Comment:
State License # shouldn't be removed, right? This issue there was that it's not a required field.